web/templates/play/show.html.eex

```diff
@@ -1,2 +1,4 @@
 <%= render "_modal_keybinds.html" %>
 <div id="board-viewer" class="play-show"></div>
+
+<%= link "Explodey View", to: skin_path(@conn, :show, @game), class: "btn btn-default btn-xs view-btn" %>
```

Comment:
It isn't necessary to change once you're already watching, i would prefer to only keep the link on index or edit pages, were the standard viewing links already exist.